deployment/src/kubernetes/Backup.md

```diff
@@ -1,305 +0,0 @@
-# Kubernetes Backup Of Ocelot.Social
```

Comment:
Are you sure that we shall remove all the manual docu?

Reply:
Debug section
-> ls & bash command auf docker container